vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -111,9 +111,6 @@ function Get-FilesToCacheOnVHD {
             "https://acs-mirror.azureedge.net/csi-proxy/v0.2.2/binaries/csi-proxy-v0.2.2.tar.gz"
         );
         "c:\akse-cache\win-k8s\"      = @(
-            "https://acs-mirror.azureedge.net/kubernetes/v1.16.10-hotfix.20200817/windowszip/v1.16.10-hotfix.20200817-1int.zip",
-            "https://acs-mirror.azureedge.net/kubernetes/v1.16.13-hotfix.20200917/windowszip/v1.16.13-hotfix.20200917-1int.zip",
-            "https://acs-mirror.azureedge.net/kubernetes/v1.16.15-hotfix.20200903/windowszip/v1.16.15-hotfix.20200903-1int.zip",
```

Comment:
I think that we should still keep these 3 versions in case users do not upgrade cluster but only update node image.
I think that we can remove v1.16.10 in next iteration.

Reply:
I raised a disscussion with PM, let's keep it until we have a final decision.